src/internet_identity_interface/src/lib.rs

```diff
@@ -227,6 +228,37 @@ pub struct ArchiveConfig {
     pub entries_fetch_limit: u16,
 }
 
+#[derive(Clone, CandidType, Deserialize, Eq, PartialEq, Debug)]
+pub struct ActiveAnchorStatistics {
+    // Stats for the last completed collection period for daily and monthly active anchors
+    pub completed: CompletedActiveAnchorStats,
+    // ongoing periods for daily and monthly active anchors
+    pub ongoing: OngoingActiveAnchorStats,
+}
+
+#[derive(Clone, CandidType, Deserialize, Eq, PartialEq, Debug)]
+pub struct CompletedActiveAnchorStats {
+    pub daily_active_anchors: Option<ActiveAnchorCounter>,
+    pub monthly_active_anchors: Option<ActiveAnchorCounter>,
+}
+
+#[derive(Clone, CandidType, Deserialize, Eq, PartialEq, Debug)]
+pub struct OngoingActiveAnchorStats {
+    // Ongoing active anchor counter for
+    pub daily_active_anchors: ActiveAnchorCounter,
+    // Monthly active users are collected using 30 day sliding windows.
+    // This vec contains up to 30 30-day active windows each offset by one day.
+    // The vec is sorted with the first element being the next window to reach the end of the
```

Comment:
not sure whether this file or .did-file is the source-of-truth, but also here explaining in terms of newer buckets could make things easier to understand.